src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingSubmissionAndResultBitbucketBambooIntegrationTest.java

```diff
@@ -1151,6 +1169,16 @@ private Long getParticipationIdByType(IntegrationTestParticipationType participa
         };
     }
 
+    private void mockTriggerByTypeAndParticipationId(IntegrationTestParticipationType participationType, Long participationId) throws Exception {
+        var buildPlanId = "";
+        switch (participationType) {
+            case SOLUTION -> buildPlanId = solutionProgrammingExerciseParticipationRepository.findById(participationId).orElseThrow().getBuildPlanId();
+            case TEMPLATE -> buildPlanId = templateProgrammingExerciseParticipationRepository.findById(participationId).orElseThrow().getBuildPlanId();
+            default -> buildPlanId = studentParticipationRepository.findById(participationId).orElseThrow().getBuildPlanId();
```

Comment:
Can you make this explicit and use the `case STUDENT` here so that we don't forget to change this in the future should the `IntegrationTestParticipationType` enum be changed?